templates/arista_eos_show_interfaces.textfsm

```diff
@@ -10,7 +10,7 @@ Value MTU (\d+)
 Value BANDWIDTH (\d+\s+\w+)
 
 Start
-  ^${INTERFACE}\s+is\s+${LINK_STATUS}.*protocol\s+is\s+${PROTOCOL_STATUS} 
+  ^${INTERFACE}\s+|\d+is\s+${LINK_STATUS}.*protocol\s+is\s+${PROTOCOL_STATUS}
```

Comment:
Please provide new `.raw` and `.yml` in the tests for this as well to make sure it's being properly captured and we know what the output looks like for this change as well.

Reply:
Yes, see https://github.com/networktocode/ntc-templates/pull/682